src/core/jsproxy.c

```diff
@@ -696,27 +696,39 @@ JsException_AsJs(PyObject* err)
   return hiwire_incref(js_error->js);
 }
 
+// Copied from Python 3.9
```

Comment:
```suggestion
// Copied from Python 3.9
// TODO: remove once we update to Python 3.9
```
If I understand correctly?

Reply:
Absolutely, it'll lead to a linker error otherwise.